LocalChat/LocalChat/Controllers/SettingViewController.swift

```diff
@@ -8,20 +8,21 @@
 
 import UIKit
 
-class SettingViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
-    
-    var checked = [Bool](count: 8, repeatedValue: false)
-    var checkedNumber: Int = 0
+class SettingViewController: UIViewController, UITableViewDelegate, UITableViewDataSource, UITextFieldDelegate {
+
+    var checked = [Bool](count: Constants.maxUser, repeatedValue: false)
+    var checkedNumber = 0
```

Comment:
Howerver, in case data type is implicit, specification is necessary.

// ◯ Int / UInt / NSInteger / NSUInteger these data types should be distinguished so specification is necessary
let i: Int = 0

// ◯ Float / CGFloat / Double these data types should be distinguished so specification is necessary
let p: Float = 3.14